frontend/src/libraries/explorer-wamp/details.ts

```diff
@@ -2,7 +2,7 @@ import { ExplorerApi } from ".";
 
 export interface Details {
   accountsCount: number;
-  totalNodesCount: number;
+  validatorCount: number;
```

Comment:
Following the style of the above variable name, I would call it with `s` at the end: `validatorsCount`